app/src/main/java/com/example/learingrealmandretrofit/api/ApiInterface.kt

```diff
@@ -1,31 +1,46 @@
 package com.example.learingrealmandretrofit.api
 
-import com.example.learingrealmandretrofit.objects.Card
-import com.example.learingrealmandretrofit.objects.Deck
+import com.example.learingrealmandretrofit.objects.CardParameters
+import com.example.learingrealmandretrofit.objects.DeckParameters
+import com.example.learingrealmandretrofit.objects.request.SignInRequest
+import com.example.learingrealmandretrofit.objects.request.SignInUpRequest
 import com.example.learingrealmandretrofit.objects.response.*
 import retrofit2.Call
 import retrofit2.http.*
 
 interface ApiInterface {
+    companion object {
+        private const val tokenKey: String = "X-Session-Token"
+    }
+
     @GET("cards")
-    fun getCards(): Call<CardListResponse>
+    fun getCards(@Header(tokenKey) token: String): Call<CardListResponse>
```

Comment:
Group endpoints by domain of their application or by class that they are operating with